lib/src/main/resources/md/ddm/ddm.xsd

```diff
@@ -415,6 +420,50 @@
         </xs:restriction>
     </xs:simpleType>
 
+    <!-- =================================================================================== -->
+    <xs:element name="reportNumber" substitutionGroup="dcterms:subject" type="ddm:ReportNumberType">
+        <xs:annotation>
+            <xs:documentation xml:lang="en">
+            </xs:documentation>
+        </xs:annotation>
+    </xs:element>
+
+    <xs:complexType name="ReportNumberType">
+        <xs:complexContent>
+            <xs:extension base="dc:SimpleLiteral">
+                <xs:attribute name="schemeURI" use="required" type="xs:anyURI" fixed="https://data.cultureelerfgoed.nl/term/id/abr/7a99aaba-c1e7-49a4-9dd8-d295dbcc870e">
+                    <xs:annotation>
+                        <xs:documentation>The URI of the scheme. For a SKOS scheme, the ConceptScheme in the 'skos:inScheme' value is expected</xs:documentation>
+                    </xs:annotation>
+                </xs:attribute>
+                <xs:attribute name="valueURI" use="required" type="xs:anyURI">
+                    <xs:annotation>
+                        <xs:documentation>The URI of the value given in the text-node.</xs:documentation>
+                    </xs:annotation>
+                </xs:attribute>
+                <xs:attribute name="subjectScheme" type="xs:string" fixed="ABR Rapporten" use="required">
+                    <xs:annotation>
+                        <xs:documentation>A human readable title of the scheme used.</xs:documentation>
+                    </xs:annotation>
+                </xs:attribute>
+                <xs:attribute name="reportNo" type="xs:string" use="optional">
+                    <xs:annotation>
+                        <xs:documentation>The identifying part of the report number. If the value in the text-node equals 'BAAC 123-A' the value of this attribute should be '123-A'</xs:documentation>
+                    </xs:annotation>
+                </xs:attribute>
+            </xs:extension>
+        </xs:complexContent>
+    </xs:complexType>
+
+    <!-- =================================================================================== -->
+    <xs:element name="acquisitionMethod" substitutionGroup="dcterms:subject" type="ddm:SubjectType">
+        <xs:annotation>
+            <xs:documentation xml:lang="en">Use this to describe the method of acquisition. Terms can be taken from a controlled vocabulary
+                like the ABR. See https://data.cultureelerfgoed.nl/term/id/abr/554ca1ec-3ed8-42d3-ae4b-47bcb848b238.html
+            </xs:documentation>
+        </xs:annotation>
+    </xs:element>
+
```

Comment:
I get a failure 

    org.xml.sax.SAXParseException; lineNumber: 26; columnNumber: 237; cvc-complex-type.2.4.a: Invalid content was found starting with element 'reportNumber'. One of '{"http://purl.org/dc/elements/1.1/":any}' is expected.

with 
https://github.com/DANS-KNAW-jp/easy-convert-bag-to-deposit/blob/435bbc2004277e2450114706b9a75fcf9c792f4a/src/test/scala/nl.knaw.dans.easy.bag2deposit/RewriteSpec.scala#L36
https://github.com/DANS-KNAW-jp/easy-convert-bag-to-deposit/blob/435bbc2004277e2450114706b9a75fcf9c792f4a/src/test/scala/nl.knaw.dans.easy.bag2deposit/RewriteSpec.scala#L125

Reply:
fixed my own mistakes with c5d2594e